src/mods/visual/model.ts

```diff
@@ -24,7 +24,11 @@ export async function loadVisuals(shard: Shard, userId: string, roomName: string
 		return visualsString;
 	}
 	if (payload.time && typedArrayToString(payload.time) === `${shard.time}`) {
-		return stringify(payload['*']) + stringify(payload[roomName]);
+		if (roomName === 'map') {
```

Comment:
Can you modify the `hmget` above so that it doesn't fetch `*` in the case we're fetching `map`?